src/services/inventoryService.ts

```
@@ -301,12 +302,12 @@ export const addItem = async (
             switch (typeName.substr(1).split("/")[2]) {
                 case "Sentinels":
                     // TOOD: Sentinels should also grant their DefaultUpgrades & SentinelWeapon.
-                    const sentinel = await addSentinel(typeName, accountId);
+                    const inventoryChanges = await addSentinel(typeName, accountId);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)